src/pymor/models/interface.py

```diff
@@ -205,10 +281,19 @@ def compute(self, solution=False, output=False,
             If `True`, return the model's internal state.
         output
             If `True`, return the model output.
+        solution_d_mu
+            If not `False`, either `True` to return the derivative of the model's
+            internal state w.r.t. all parameter components or a tuple `(parameter, index)`
+            to return the derivative of a single parameter component.
+        output_d_mu
+            If `True`, return the gradient of the model output w.r.t. the |Parameter|.
         solution_error_estimate
             If `True`, return an error estimate for the computed internal state.
         output_error_estimate
             If `True`, return an error estimate for the computed output.
+        output_d_mu_return_array
+            if `True`, return the output of :meth:`_compute_output_d_mu` as a |Numpy array|,
+            otherwise, return it as a dict w.r.t. to the parameters.
```

Comment:
```suggestion
            If `True`, return the output gradient as a |Numpy array|. Otherwise, return a dict of gradients for
            each |Parameter|.
```

You can't reference `_compute_output_d_mu` here as this is not an interface method.